AndelaOSP/shufflebox-frontend
# #138738207 Brownbag and Hangouts Pages

#### What does this PR do?
Add a Brownbag and Hangouts pages to the app.

#### Description of Task to be completed?
- Export common elements to a Common directory
- Create directories for the Brownbag and Hangouts directories
- Create messages, models, updates and views for both

#### How should this be manually tested?
Run the app and click on the Brownbag link on the menu; the brownbag page should load. The same applies for the hangouts link.

#### What are the relevant pivotal tracker stories?
#138738207
